docs/sdk/tutorials/importing_video_assets.md

```diff
@@ -222,6 +223,39 @@ assets = kili.append_many_to_dataset(
 )
 ```
 
+## Updating video metadata
+
+When importing a video, if you want to add or delete metadata, you must provide the entire list of metadata. This new list will replace the previous one, meaning any metadata not included in the new list will be erased. However, this does not apply to the `processingParameters` metadata, which will be retained even if they are not present in the new list.
```

Comment:
I find this sentence not so clear, especially that after this you provide an example where you update the processing parameters. I would say that they will not be modified, except if the processingParameters are explicitly provided. And maybe explain that they have to do it only if they understand what they do as it can have an impact on how the video will be managed by the application.

Reply:
Yes i've update the documentation, thanks